apps/api/src/domain/models/vocabulary-list/repositories/arango-vocabulary-list-query-repository.ts

```diff
@@ -227,16 +217,15 @@ export class ArangoVocabularyListQueryRepository implements IVocabularyListQuery
         };
 
         /**
-         * We need to figure out why we get the dreaded `write-write` conflict
-         * with arango.
-         *
-         * Note that the following query works fine when executed from the ArangoDB dashboard:
-         *
+         * Note that we use `UNION_DISTINCT` to update actions in case this event has already been seen
+         * once. We only need to open the option to `AnalyzeTermInVocabularyList`
```

Comment:
I don't know enough about the vocabulary list to really comment here, but there's no scenario where a term would be re-analyzed or the analysis updated?  This probably a nonsensical question.

Reply:
Ah, that's a good question. Currently, we don't allow overwriting an existing property:
![image](https://github.com/user-attachments/assets/2aeb2848-b18b-418b-9db8-12e688b98636)

Reply:
We probably want to have either a:
`RemoveVocabularyListFilterProperty`
or
`UpdateVocabularyListFilterProperty`
if we hit that situation, or maybe
`RemoveEntryFromVocabularyList`